app/assets/javascripts/cable.js

```diff
@@ -0,0 +1,4 @@
++function() {
```

Comment:
```suggestion
+function () {
```

Solo para acordarme.